.github/workflows/aws-deploy.yaml

```diff
@@ -0,0 +1,128 @@
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_KEY }}
+          aws-region: ${{ inputs.AWS_REGION }}
+
+      # https://github.com/aws-actions/amazon-ecr-login
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      # https://github.com/docker/build-push-action
+      # https://github.com/docker/build-push-action/blob/master/docs/advanced/cache.md
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.login-ecr.outputs.registry }}/${{ inputs.ECR_REPOSITORY }}:${{ github.sha }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+      # https://github.com/aws-actions/amazon-ecs-render-task-definition
+      - name: Fill in the new image ID in the Amazon ECS task definition
+        id: task-def
+        uses: aws-actions/amazon-ecs-render-task-definition@v1
+        with:
+          task-definition: ${{ inputs.ECS_TASK_DEFINITION }}
+          container-name: ${{ inputs.CONTAINER_NAME }}
+          image: ${{ steps.login-ecr.outputs.registry }}/${{ inputs.ECR_REPOSITORY }}:${{ github.sha }}
+
+      # https://github.com/aws-actions/amazon-ecs-deploy-task-definition
+      - name: Deploy Amazon ECS task definition
+        uses: aws-actions/amazon-ecs-deploy-task-definition@v1
+        with:
+          task-definition: ${{ steps.task-def.outputs.task-definition }}
+          service: ${{ inputs.ECS_SERVICE }}
+          cluster: ${{ inputs.ECS_CLUSTER }}
+          wait-for-service-stability: true
+
+      # https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files
+      - id: set_var
+        run: |
+          content=`aws --region eu-west-2 ecs describe-services --services admin-dashboard --cluster hl-stage`
+          # the following lines are only required for multi line json
+          content="${content//'%'/'%25'}"
+          content="${content//$'\n'/'%0A'}"
+          content="${content//$'\r'/'%0D'}"
+          # end of optional handling for multi line json
+          echo "::set-output name=packageJson::$content"
+      - id: status
+        run: |
+          if [ "${{fromJson(steps.set_var.outputs.packageJson).services[0].runningCount }}" != "1" ]; then echo "SLACK_COLOR=#ff00ff" >> $GITHUB_ENV ; else echo "SLACK_COLOR=green" >> $GITHUB_ENV ; fi
+      - run: |
+          echo "${{ env.SLACK_COLOR }}"
+
+        # https://github.com/marketplace/actions/slack-notify
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_CHANNEL: Ivanov Konstantin
+          SLACK_COLOR: ${{ env.SLACK_COLOR }} # or a specific color like 'good' or '#ff00ff'
+          SLACK_MESSAGE: ${{ github.event.head_commit.message }}
```

Comment:
Can you do it bit more user friendly with a mesasge like "Releasing XYZ to [environment] bla bla .. commit: 1234.."

Reply:
![Screenshot from 2022-05-06 16-52-46](https://user-images.githubusercontent.com/103993034/167146778-79b2c2a6-5a43-4d24-9d69-9265f4af2993.png)

This form already contains some text, maybe it is enough

Reply:
oh, that looks goood